README.md

```diff
@@ -164,6 +165,23 @@ You are able to use some custom headers in your request to gain more control ove
 
 By doing this you are now able to change those values using a custom header. This can help you with easier authentication or retrieving the userId from a `cognitoAuthenticationProvider` value.
 
+## Environment variables
+You are able to use environmnet variables to customize identity params in event context.
+
+| Environment Variable | Event key |
+|----------------------|-----------|
+| cognito-identity-id | event.requestContext.identity.cognitoIdentityId |
+| cognito-authentication-provider | event.requestContext.identity.cognitoAuthenticationProvider |
```

Comment:
I cannot see `cognito-identity-id` and `cognito-authentication-provider` in `src/createLambdaProxyContext.js`

Also, can they be in CAPS_SNAKE_CASE too ?

Reply:
Ohh, my bad. I think I forgot to delete rows after copy paste